docs/en/sql-reference/functions/tuple-map-functions.md

```diff
@@ -429,6 +429,38 @@ Result:
 │ {}                         │
 └────────────────────────────┘
 ```  
+
+
+## mapFromArrays {#mapFromArrays}  
+  
```

Comment:
Needs a description on l. 435 what the function actually does, e.g. `Merges an Array of keys and an Array of values into a Map.`